cmd/kube-burner/kube-burner.go

```diff
@@ -190,6 +193,81 @@ func destroyCmd() *cobra.Command {
 	return cmd
 }
 
+func measureCmd() *cobra.Command {
+	var uuid string
+	var rawNamespaces string
+	var rawLabels string
+	var configFile string
+	var userMetadata string
+	var indexer *indexers.Indexer
+	var labelPairs []string
+	metadata := make(map[string]interface{})
+	cmd := &cobra.Command{
+		Use:   "measure",
+		Short: "Take measurements for a given set of resources without running workload",
+		PostRun: func(cmd *cobra.Command, args []string) {
+			log.Info("👋 Exiting kube-burner ", uuid)
+		},
+		Args: cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			f, err := util.ReadConfig(configFile)
+			if err != nil {
+				log.Fatalf("Error reading configuration file %s: %s", configFile, err)
+			}
+			configSpec, err := config.Parse(configFile, f)
+			if err != nil {
+				log.Fatal(err.Error())
+			}
+			if len(configSpec.Jobs) > 0 {
+				log.Fatal("No jobs are allowed in a measure subcommand config file")
+			}
+			if configSpec.GlobalConfig.IndexerConfig.Type != "" {
+				indexerConfig := configSpec.GlobalConfig.IndexerConfig
+				log.Infof("📁 Creating indexer: %s", indexerConfig.Type)
+				indexer, err = indexers.NewIndexer(indexerConfig)
+				if err != nil {
+					log.Fatalf("%v indexer: %v", indexerConfig.Type, err.Error())
+				}
+			}
+			if userMetadata != "" {
+				metadata, err = util.ReadUserMetadata(userMetadata)
+				if err != nil {
+					log.Fatalf("Error reading provided user metadata: %v", err)
+				}
+			}
+			labels := make(map[string]string)
+			if rawLabels != "" {
+				labelPairs = strings.Split(rawLabels, ",")
+			}
+			for _, pair := range labelPairs {
+				parts := strings.SplitN(pair, "=", 2)
+				if len(parts) == 2 {
+					labels[parts[0]] = parts[1]
+				} else {
+					log.Fatalf("Invalid label format: %s", pair)
+				}
+			}
```

Comment:
This code block can be greatly reduced by using labels.Parse from the package "k8s.io/apimachinery/pkg/labels"

```suggestion
            if selector != ""{
                labelSelector, err = labels.Parse(selector)
                etc...
```

Reply:
Changed as suggested